src/main/java/step2/domain/LottoNumber.java

```diff
@@ -39,6 +50,14 @@ public int hashCode() {
 
     @Override
     public int compareTo(LottoNumber other) {
-        return this.lottoNumber - other.lottoNumber;
+        if (this.lottoNumber > other.lottoNumber) {
+            return 1;
+        }
+
+        if (this.lottoNumber < other.lottoNumber) {
+            return -1;
+        }
+
+        return 0;
```

Comment:
compareTo 오버플로우 방지 👍 
다음과 같이도 사용할 수 있으니 참고해주시면 좋을 것 같아요!
```suggestion
        return Integer.compare(this.lottoNumber, other.lottoNumber);
```

Reply:
꿀팁 감사합니다